.gitignore

```
@@ -33,3 +33,10 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+
+/public/packs
+/public/packs-test
+/node_modules
```

こっちが正しいです！（もしかしたらnode_modules/**/*かも？）
```suggestion
/node_modules/**/*
```

---

ありがとうございます！試してみます！

---

node_modulesに関して解決しました！ありがとうございます😭
gitで無視するべきファイルなのにも関わらず、gitに追跡されていたという解釈であってますでしょうか？

---

その通りです💡
node_modulesには`package.json`で呼び出してるJSのパッケージと依存関係が丸ごと格納されてるので、gitに上げるべきではないものです！
Railsでは元々デフォルトの.gitignoreに`node_modules`が含まれてたはずなんですが、どっかで消えてしまってたりしたのかもですね。。

---

> .gitignoreにnode_modulesが含まれてたはずなんですが、

コミットログ見たら最初から入ってなかったですね！
これは多分Railsのバージョンによるものだと思います💡

今までRailsではJSの読み込みにNode.jsを使っていたんですが、今回使っているRails7では、デフォルトではNode.jsを使わない設定になっています。
そういう背景からNode.jsに依存したファイル群(`package.json`, `yarn lock`, `node_modules`あたり)は存在しない前提なので、`.gitignore`にも含まれていなかった → 今回のPRでNode.jsを入れたので、`node_modules`が差分に混ざったんだと思います！

Node.js周りの知識は、ある程度歴史的背景も知っておいた方が頭に入ってきやすいと思うので、この間紹介した「りあクト！」がおすすめです！
https://booth.pm/ja/items/2368045